Ast/src/StringUtils.cpp

```diff
@@ -272,6 +272,14 @@ std::string escape(std::string_view s)
             case '\"':
                 r += '\"';
                 break;
+            // INTERP CODE REVIEW: This is going to apply it to all escaped strings, not just
+            // interpolated ones. Is that acceptable, or should this be split into two functions/a toggle?
```

Comment:
Bumping this one

Reply:
Yeah I don't think we should do this unconditionally, in particular for `{` it will look weirdly asymmetric. Maybe a boolean parameter?